assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -80,12 +85,104 @@ export const actions = {
 			type: RECEIVE_ACCOUNTS_PROPERTIES_PROFILES_FAILED,
 		};
 	},
+
+	/**
+	 * Creates a new Analytics account.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} args              Argument params.
+	 * @param {string} args.accountName  Google Analytics account name.
+	 * @param {string} args.propertyName Google Analytics property name.
+	 * @param {string} args.profileName  Google Analytics profile name.
+	 * @param {string} args.timezone     Google Analytics timezone.
+	 * @return {Function} Generator function action.
+	 */
+	*createAccount( { accountName, propertyName, profileName, timezone } ) {
+		invariant( accountName, 'accountName is required.' );
+		invariant( propertyName, 'propertyName is required.' );
+		invariant( profileName, 'profileName is required.' );
+		invariant( timezone, 'timezone is required.' );
+
+		const accountTicket = yield actions.fetchCreateAccount( { accountName, propertyName, profileName, timezone } );
+
+		return actions.receiveCreateAccount( { accountTicket } );
```

Comment:
Let's move this call directly into `fetchCreateAccount()` (after retrieving the `response`), like it is handled elsewhere. This will also simplify this action creator to only call the `yield actions.fetchCreateAccount` above.

Reply:
Moved in https://github.com/google/site-kit-wp/pull/1390/commits/acb3c3134ca03c53c4058b593fbae64dd371f123, please review.